LoRaEngine/modules/LoRaBasicsStationModule/Dockerfile.amd64

```
@@ -1,6 +1,6 @@
 #This docker builds a container for the LoRa Basics station on amd64 architecture
 
-FROM amd64/debian:buster as build
+FROM lorae2etestsacr.azurecr.io/amd64/debian:buster as build
```

Instead of hardcoding this, which is not accessible by anyone outside of our team, let's do it differently.

```suggestion
ARG SOURCE_CONTAINER_REGISTRY_ADDRESS=docker.io
FROM $SOURCE_CONTAINER_REGISTRY_ADDRESS/amd64/debian:buster
```

Then when we do the buildx bake in the ci we should override as follows
```bash
docker buildx bake --set *.args.SOURCE_CONTAINER_REGISTRY_ADDRESS=${{ secrets.CI_ACR_NAME }}.azurecr.io
```

---

that's a great idea!